src/fluree/db/json_ld/commit_data.cljc

```diff
@@ -494,11 +501,20 @@
                                 (flake/create new-issuer-sid const/$xsd:anyURI issuer-iri const/$xsd:string t true nil)])))
           message-flakes (when message
                            [(flake/create t const/$_commit:message message const/$xsd:string t true nil)])
+          default-ctx-flakes (when-let [default-ctx-iri (:id defaultContext)]
+                               (if-let [default-ctx-id (<? (dbproto/-subid db default-ctx-iri))]
+                                 [(flake/create t const/$_ledger:context default-ctx-id const/$xsd:anyURI t true nil)]
```

Comment:
It shouldn't since that's just a context, not a _default_ context. I don't think it would given the different key naming.

Reply:
I think this is a good call where you have it. 

I think we could go any way, including not storing it as a Flake at all. I think the balance is thinking if it will be useful in a query for people to ask questions.

I think the most important question will be when did the context change and what are the addresses... so having flakes to answer this I think would be useful:
```
{:select [?t ?ctx-id ?ctx-address]
 :where [[?t :f/defaultContext ?ctx-id]
               [?ctx-id :f/address ?ctx-address]]}
```

Which is exactly what you have.

You are also linking the ?ctx-id to every `t`. That might be a bit excessive, but I can see it being useful for asking for it at a specific `t` (e.g. t=42 here) with:
```
{:select [?ctx-id ?ctx-address]
 :where [[42 :f/defaultContext ?ctx-id]
               [?ctx-id :f/address ?ctx-address]]}
```

Even perhaps excessive, it is a relatively compact amount of information so if not needed, its not like we are paying a super high price.

So I like your decision here. I think we'll get better feedback once we have more users and can change course if necessary then.